lib/qtcamconfig.cpp

```diff
@@ -118,9 +118,9 @@ QtCamConfig::QtCamConfig(QObject *parent) :
   d_ptr(new QtCamConfigPrivate) {
 
 #ifdef QT4
-  d_ptr->model = QtMobility::QSystemDeviceInfo().model().toLower();
+  d_ptr->model = QtMobility::QSystemDeviceInfo().productName().toLower();
```

Comment:
N9/N950 are not affected, since there are no directories for them under configs/, and resolutions.ini seem to overwrite one another IIUC: https://github.com/foolab/cameraplus/blob/a6a565c2ba56dbfc8857dcfab6943f1ba050795f/debian/install#L10